client.js

```diff
@@ -156,34 +391,131 @@ bot.on('message', function (message) {
+    const historical_pricing = await fetchWikiPricing(item, server)
+
+    const fields = []
+
+    fields.push({
+        name: formattedPrice || 'No Price Listed',
+        value: `${formatCapitalCase(item)}`,
+        inline: false
+    })
+
+    if (historical_pricing) {
+        fields.push({
+            name: `Historical Pricing Data`,
+            value: historical_pricing,
+            inline: false
+        })
     }
-    msg += `***${fullAuction}***\nTo snooze notifications for this watch for the next hour, click 💤. To remove it, click ❌. To ignore auctions by this seller, click 🔕.`
-    
-    if (bot.users.cache.get(userID.toString()) === undefined) {
-        console.log('sending msg to user ', userID)
-        bot.guilds.cache.get(GUILD).members.fetch(userID.toString()).then((res)=>{res.send(msg)}).catch((err)=> {console.log(err)});
+
+    let msg = new Discord.MessageEmbed()
+        .setColor(SERVER_COLOR[server])
+        .setImage(url === `https://i.imgur.com/wXJsk7Y.png` ? null : url)
+        .setTitle(`${formatCapitalCase(item)}`)
+        .setAuthor('Watch Notification', url, `https://wiki.project1999.com${wiki_url[item]}`)
+        .setDescription(`**${seller}** is currently selling **${formatCapitalCase(item)}** ${price ? 'for **' + price + 'pp**' : ''} on Project 1999 **${formatCapitalCase(server)}** server. \n\n\`\`${removeLogTimestamp(fullAuction)}\`\``)
+        .addFields(fields)
+        .setFooter(`To snooze this watch for 6 hours, click 💤\nTo end this watch, click ❌\nTo ignore auctions by this seller, click 🔕\nTo extend this watch, click ♻\nWatch expires ${moment(timestamp).add(7, 'days').fromNow()}`)
+        .setTimestamp()
+    if (bot.users.cache.get(user.toString()) === undefined) {
+        bot.guilds.cache.get(GUILD).members.fetch(user.toString())
+            .then((res)=>{
+                res.send(msg)
+                    .then(message => {
+                        message
+                        .react('💤')                     // for "snooze watch"
+                        .then(() => message.react('❌')) // for "delete watch"
+                        .then(() => message.react('🔕')) // for "silence seller"
+                        .then(() => message.react('♻'))  // for "extend watch"
+                        .then(() => {
+                            const react_filter = (reaction, user) => {
+                                return reaction.emoji.name === '💤' || reaction.emoji.name === '❌' || reaction.emoji.name === '🔕' || reaction.emoji.name === '♻';
+                            }
+                            const collector = message.createReactionCollector(react_filter, { time: 1000 * 60 * 60 * 24 , dispose: true});
+                            collector.on('collect', (reaction, user) => {
+                                if (user.bot) return;
+                                switch (reaction.emoji.name) {
+                                    case '💤': //alt: 🔕
```

Comment:
That "alt" isn't valid anymore as you've chosen to use that emoji for "silence seller", so having it listed here confused me for a moment

Reply:
Oops, my bad.